### PR TITLE
Add user cookie

### PR DESCRIPTION
**Pull request by 9shikixp**
Userがログインできるようになりました(Cookieを使いました)。

### CODE REVIEW

File: Katan/config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
+  get  'login' => 'sessions#new'
+  post 'login' => 'sessions#create'
+  delete 'logout' => 'sessions#destroy'
+
   get 'wssample/index'
   root 'tops#index'
+
+
```
Comment (ctare) [inline Katan/config/routes.rb:9]: **[rubocop]**
- Extra blank line detected.
- Line is too long. [84/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

File: Katan/test/controllers/sessions_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class SessionsControllerTest < ActionController::TestCase
```
Comment (ctare) [inline Katan/test/controllers/sessions_controller_test.rb:3]: **[rubocop]**
- Missing top-level class documentation comment.

File: Katan/test/controllers/sessions_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  test "should get new" do
```
Comment (ctare) [inline Katan/test/controllers/sessions_controller_test.rb:4]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 4]
```diff
@@ -11,3 +11,69 @@
...
 
+ActiveRecord::Schema.define(version: 20160902123105) do
+
+  create_table "building_types", force: :cascade do |t|
+    t.string   "name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "game_buildings", force: :cascade do |t|
```
Comment (ctare) [inline Katan/db/schema.rb:22]: **[rails_best_practices]**
- always add db index (game_buildings => [user_id]) [:ref](http://rails-bestpractices.com/posts/2010/07/24/always-add-db-index/)

File: Katan/test/controllers/sessions_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+    get :new
+    assert_response :success
+  end
+
```
Comment (ctare) [inline Katan/test/controllers/sessions_controller_test.rb:8]: **[rubocop]**
- Extra empty line detected at class body end.

File: Katan/db/schema.rb [hunk 6]
```diff
@@ -11,3 +11,69 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+ActiveRecord::Schema.define(version: 20160902123105) do
```
Comment (ctare) [inline Katan/db/schema.rb:14]: **[rubocop]**
- Separate every 3 digits in the integer portion of a number with underscores(_). [:ref](https://github.com/bbatsov/ruby-style-guide#underscores-in-numerics)

File: Katan/db/schema.rb [hunk 7]
```diff
@@ -11,3 +11,69 @@
...
+
```
Comment (ctare) [inline Katan/db/schema.rb:15]: **[rubocop]**
- Extra empty line detected at block body beginning.

File: Katan/db/schema.rb [hunk 8]
```diff
@@ -11,3 +11,69 @@
...
+  create_table "building_types", force: :cascade do |t|
```
Comment (ctare) [inline Katan/db/schema.rb:16]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 9]
```diff
@@ -11,3 +11,69 @@
...
+    t.string   "name"
```
Comment (ctare) [inline Katan/db/schema.rb:17]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 10]
```diff
@@ -11,3 +11,69 @@
...
+    t.datetime "created_at", null: false
```
Comment (ctare) [inline Katan/db/schema.rb:18]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 11]
```diff
@@ -11,3 +11,69 @@
...
+    t.datetime "updated_at", null: false
```
Comment (ctare) [inline Katan/db/schema.rb:19]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb (hunk 4 above)
Comment (ctare) [inline Katan/db/schema.rb:22]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 12]
```diff
@@ -11,3 +11,69 @@
...
+  end
+
+  create_table "game_buildings", force: :cascade do |t|
+    t.string   "type"
```
Comment (ctare) [inline Katan/db/schema.rb:23]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 13]
```diff
@@ -11,3 +11,69 @@
...
+    t.integer  "user_id"
```
Comment (ctare) [inline Katan/db/schema.rb:24]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 14]
```diff
@@ -11,3 +11,69 @@
...
+    t.datetime "created_at", null: false
```
Comment (ctare) [inline Katan/db/schema.rb:25]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 15]
```diff
@@ -11,3 +11,69 @@
...
+    t.datetime "updated_at", null: false
```
Comment (ctare) [inline Katan/db/schema.rb:26]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 16]
```diff
@@ -11,3 +11,69 @@
...
+  end
+
+  create_table "game_fields", force: :cascade do |t|
```
Comment (ctare) [inline Katan/db/schema.rb:29]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 17]
```diff
@@ -11,3 +11,69 @@
...
+    t.integer  "gameresource_id"
```
Comment (ctare) [inline Katan/db/schema.rb:30]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 18]
```diff
@@ -11,3 +11,69 @@
...
+    t.integer  "number"
```
Comment (ctare) [inline Katan/db/schema.rb:31]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 19]
```diff
@@ -11,3 +11,69 @@
...
+    t.datetime "created_at",      null: false
```
Comment (ctare) [inline Katan/db/schema.rb:32]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 20]
```diff
@@ -11,3 +11,69 @@
...
+    t.datetime "updated_at",      null: false
```
Comment (ctare) [inline Katan/db/schema.rb:33]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 21]
```diff
@@ -11,3 +11,69 @@
...
+  end
+
+  create_table "game_intersections", force: :cascade do |t|
```
Comment (ctare) [inline Katan/db/schema.rb:36]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 22]
```diff
@@ -11,3 +11,69 @@
...
+    t.integer  "gamebuilding_id"
```
Comment (ctare) [inline Katan/db/schema.rb:37]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 23]
```diff
@@ -11,3 +11,69 @@
...
+    t.datetime "created_at",      null: false
```
Comment (ctare) [inline Katan/db/schema.rb:38]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 24]
```diff
@@ -11,3 +11,69 @@
...
+    t.datetime "updated_at",      null: false
```
Comment (ctare) [inline Katan/db/schema.rb:39]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 25]
```diff
@@ -11,3 +11,69 @@
...
+  end
+
+  create_table "game_maps", force: :cascade do |t|
```
Comment (ctare) [inline Katan/db/schema.rb:42]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 26]
```diff
@@ -11,3 +11,69 @@
...
+    t.integer  "gamefield_id"
```
Comment (ctare) [inline Katan/db/schema.rb:43]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 27]
```diff
@@ -11,3 +11,69 @@
...
+    t.datetime "created_at",   null: false
```
Comment (ctare) [inline Katan/db/schema.rb:44]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 28]
```diff
@@ -11,3 +11,69 @@
...
+    t.datetime "updated_at",   null: false
```
Comment (ctare) [inline Katan/db/schema.rb:45]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 29]
```diff
@@ -11,3 +11,69 @@
...
+  end
+
+  create_table "game_resources", force: :cascade do |t|
```
Comment (ctare) [inline Katan/db/schema.rb:48]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 30]
```diff
@@ -11,3 +11,69 @@
...
+    t.string   "type"
```
Comment (ctare) [inline Katan/db/schema.rb:49]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 31]
```diff
@@ -11,3 +11,69 @@
...
+    t.datetime "created_at", null: false
```
Comment (ctare) [inline Katan/db/schema.rb:50]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 32]
```diff
@@ -11,3 +11,69 @@
...
+    t.datetime "updated_at", null: false
```
Comment (ctare) [inline Katan/db/schema.rb:51]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 33]
```diff
@@ -11,3 +11,69 @@
...
+  end
+
+  create_table "game_sides", force: :cascade do |t|
```
Comment (ctare) [inline Katan/db/schema.rb:54]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 34]
```diff
@@ -11,3 +11,69 @@
...
+    t.integer  "gamebuilding_id"
```
Comment (ctare) [inline Katan/db/schema.rb:55]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

File: Katan/db/schema.rb [hunk 35]
```diff
@@ -11,3 +11,69 @@
...
+    t.datetime "created_at",      null: false
```
Comment (ctare) [inline Katan/db/schema.rb:56]: **[rubocop]**
- Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

**Comment by projectormato** [comment]
すごいすごい！

**Pull request merged**